### PR TITLE
Line item variant titles

### PR DESCRIPTION
**Pull request by TimeBones**
[HDLS-205] - Add Line Item Product Title

**CHANGES**
* useVariant hook added
* line items show variants
* If stock isn't tracked an inventory issue will not be raised

### CODE REVIEW

File: one_click/server/index.js [hunk 1]
```diff
@@ -104,9 +104,9 @@ app.get('/inventory', async ( req, res ) => {
   })
   const invResult = await invRequest.json();
   const invList = invResult.data.map((item) => {
-      return { id:    item.platform_id,
-               stock: item.inventory_quantity,
-               name:  item.name 
+      return { id:        item.platform_id,
+               stock:     item.inventory_quantity,
+               tracking:  item.inventory_tracking_service
```
Comment (CalinR) [inline one_click/server/index.js:109]: I think this should also check to see if the variant allows backordering as well. It should be called allow_backorder in the response we get back from the products endpoint

**Review by CalinR — CHANGES_REQUESTED** [review]

File: one_click/server/index.js (hunk 1 above)
Comment (TimeBones) [reply one_click/server/index.js:109]: Not getting allow_backorder from big commerce response, going to hold off until we can test that.

File: one_click/client/components/OneClickLayout/components/LineItems/LineItems.jsx [hunk 2]
```diff
@@ -35,6 +37,7 @@ LineItems.propTypes = {
   removeLineItem: PropTypes.func,
 };
 
+const handleVariants = useVariants();
```
[the remark this answers is not in the document]
Comment (TimeBones) [reply one_click/client/components/OneClickLayout/components/LineItems/LineItems.jsx]: Fixed.

File: one_click/client/hooks/useVariants.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+const useVariants = () => {
```
[the remark this answers is not in the document]
Comment (TimeBones) [reply one_click/client/hooks/useVariants.js:1]: Refactored hook and added map to description in product + inventory item.

**Pull request closed without merging**